pkg/operator/ceph/object/bucket/provisioner.go

```diff
@@ -201,10 +201,7 @@ func (p Provisioner) Delete(ob *bktv1alpha1.ObjectBucket) error {
 		return err
 	}
 
-	err = p.deleteOBCResource(p.bucketName)
```

Comment:
Actually, in this place we still want to return the error instead of suppressing it.